src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
+
+
+
+
+
+class Content extends Component{
+    constructor(props){
+        super(props);
+        this.state={};
+
+        this.handlename=this.handlename.bind(this);
+    }
+
+
+
+    handlename(props){
+        alert(renderboth)
+    }
```

Comment:
should be camel case and these local methods dont have a need for a props argmunent so probably dont need it. Also to set up the better logic you should have a random name method above this one and have this one then update the state with the result so it gets displayed on the screen kinda like below
```suggestion
   generateRandomName() {
       return this.firstNames[Math.floor(Math.random()* this.firstNames.length)] + ' ' + 
         this.lastNames[Math.floor(Math.random()* this.lastNames.length)]
   }
   
   handleName(){
        this.setState({
          randomName: this.generateRandomName()
        })
    }
```